app/views/shared/form/_required_field_legend.html.erb

```diff
@@ -0,0 +1,3 @@
+<span class="text-sm text-slate-500 dark:text-slate-300"><span class="text-red-500 dark:text-red-500">*
```

Comment:
Use `<span class="text-red-500 dark:text-red-500" aria-hidden="true">*</span>` for the asterisk so screen readers don't read it as "star".

Reply:
Updated in 721c9ad0ea656fdbd00790384833751826d7e573